playground/orleans/OrleansClient/OrleansClient.csproj

```diff
@@ -16,6 +16,9 @@
     <AspireProjectOrPackageReference Include="Aspire.Azure.Data.Tables" />
     <ProjectReference Include="..\OrleansContracts\OrleansContracts.csproj" />
     <ProjectReference Include="..\OrleansServiceDefaults\OrleansServiceDefaults.csproj" />
+
+    <!-- Component Governance -->
+    <PackageReference Include="Azure.Core" />
```

Comment:
Can we use this feature instead? https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management#transitive-pinning

That way we don't need PackageReference in all these projects.

Reply:
Looks great, will do that.